### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by ivandulko**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/ivandulko/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
+
+**Thanks for letting us show you around!**
+
+What you’re seeing here is a GitHub pull request with the code diffs
+showing by default. Feel free to look around.
+
```
Comment (ivandulko) [inline ReadMe.md:7]: Testing comments

Comment (ivandulko) [inline ReadMe.md:7]: '!fix'

Comment (ivandulko) [inline ReadMe.md:7]: !fix

Comment (ivandulko) [inline ReadMe.md:7]: !resolve !fixed !resolved !completed

File: ReadMe.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+line of code in the diff. You can also switch to the conversation
+to see comments on the whole pull request.
+
+Line notes have flags to indicate what is going on in the code,
+too. Flags are as follows:
+
+`['!fix', '!resolve']`
+- Something in the code needs to be reviewed and fixed before
+  the pull request is merged.
+
```
Comment (ivandulko) [inline ReadMe.md:18]: !fix

Comment (ivandulko) [inline ReadMe.md:18]: !fixed

File: ReadMe.md (hunk 1 above)
Comment (ivandulko) [inline ReadMe.md:7]: !resolved

**Pull request merged**